app/src/main/res/layout/activity_bmicalc.xml

```diff
@@ -4,25 +4,25 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/home_page_health_btn_background"
+    android:background="@color/colorBackgroundHome"
     tools:context=".BMICalc">
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/layoutHeaderAccountMain"
         android:layout_width="432dp"
         android:layout_height="wrap_content"
-        android:background="@drawable/home_page_header_background"
+        android:background="@drawable/home_page_health_btn_background"
         android:padding="10dp"
         app:layout_constraintTop_toTopOf="parent"
-        tools:layout_editor_absoluteX="-11dp"
-        tools:ignore="MissingConstraints">
+        tools:ignore="MissingConstraints"
```

Comment:
What is the reason for this? You've been in the code more than I so I'll defer to you, but everything I've watched emphasized the importance of constraints, unless this is not doing what it sounds like.

Reply:
It is ignoring one missing constraint on the Main Physical page that threw my UI all out of wack.